kotlin-runtime/ftl-generator/pom.xml

```diff
@@ -89,4 +89,27 @@
             </plugin>
         </plugins>
     </build>
+
+    <profiles>
+        <profile>
+            <id>release</id>
+            <build>
+                <plugins>
+                    <plugin>
+                        <groupId>org.apache.maven.plugins</groupId>
+                        <artifactId>maven-jar-plugin</artifactId>
+                        <version>3.3.0</version>
+                        <configuration>
+                            <excludes>
+                                <exclude>**/*.jar</exclude>
```

Comment:
I might be misunderstanding, but don't we want the opposite of this? We want to _only_ release `ftl-generator-X.Y.Z-jar-with-dependencies.jar`, because it's the uber JAR.

Reply:
ah sorry should've marked this PR as wip since i'm basically just using this remote branch to trigger github actions 

yes this was totally broken :) i'm playing around with different maven plugins to try and exclude the regular/non-assembly JAR from the release — this attempt aimed to exclude everything ending in `.jar` and then add back the specific suffixes that we want (-sources, -javadoc, -with-dependencies) but this isn't the right way to accomplish that